DQM/Integration/python/clients/castor_dqm_sourceclient-live_cfg.py

```diff
@@ -2,7 +2,8 @@
 import FWCore.ParameterSet.Config as cms
 import sys
 
-process = cms.Process("CASTORDQM")
+from Configuration.Eras.Era_Run3_cff import Run3
+process = cms.Process("CASTORDQM", Run3)
```

Comment:
out of curiosity, what's the point of this client if CASTOR doesn't exist in Run3?

Reply:
nothing... I'll remove it instead of trying to fix it with eras